src/mapping.jl

```diff
@@ -61,7 +61,7 @@ function cleanmapping(mapping::Dict)
         if haskey(aesthetic_aliases, key)
             key = aesthetic_aliases[key]
         elseif !in(key, fieldnames(Aesthetics))
-            warn("$(string(key)) is not a recognized aesthetic. Ignoring.")
+            @warn "$(string(key)) is not a recognized aesthetic. Ignoring."
```

Comment:
interpolations is, or it least was IIRC, slow.  so in cases like this i believe `string(key, " is not a recognized aesthetic. Ignoring.")` is preferred.

Reply:
I don't think interpolation speed has changed, but in this case I prefer less noise over performance as the warning isn't actually called anywhere near a hot loop.  I now see that it would be better to have just write: `@warn "$key is not a recognized aesthetic. Ignoring."`

Reply:
But if you prefer performance, I'll update the PR.